src/test_utils/enzyme_helpers.js

```diff
@@ -0,0 +1,24 @@
+import { mount } from "enzyme";
+import Button from "../primitives/Button"
+
+const mountWithHelpers = (element) => {
+  let component = mount(element)
+
+  component.clickButton = function(buttonText) {
```

Comment:
[Ruby's spoiled me](https://www.rubydoc.info/github/teamcapybara/capybara/master#the-dsl). (=